rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3FilterHandler.java

```diff
@@ -126,4 +154,20 @@ private boolean isAcceptedAddress(final InetAddress address) {
         }
     }
 
+    private boolean isIpAddress(String address) {
+        boolean isIPv4 = false;
+        boolean isIPv6 = false;
+
+        if (address != null) {
```

Comment:
minor thing, but I like early exits that reduce code nesting, in this case you could return false `if (address == null)`